### DISCUSSION TITLE
Strengthening UpdateOperation feature for dynamic pricing

### DISCUSSION THREAD

**Discussion by pedroblancopascual-accenture** [Shopify/function-examples#529] [Feedback: Cart Transform API]
Hello everyone,

I am analyzing the feasibility of the UpdateOperation functionality to achieve a dynamic price on a base product, depending on the add-ons that have been incorporated.

The functionality is based on the fact that the updated price is stored in a cart item property, so a user with frontend knowledge could modify that amount. How can this process be strengthened?

**Comment by InesFCB** [comment]
That is also a concern that I've found, is it possible to check into that?
